Source/Scenes/Overworld.cs

```diff
@@ -199,8 +215,59 @@ public Overworld(bool startOnLastSelected)
 		}
 
 		cameraCloseUpEase = 1.0f;
+
+		entries = GetCurrentModEntries();
+	}
+	#endregion
+
+	#region Overworld Methods
+	public List<Entry> GetCurrentModEntries()
+	{
+		List<Entry> entriesTemp = [];
+
+		// We treat the vanilla "mod" as a catch-all option (because it only has one level anyway). 
+		// It will return every available item.
```

Comment:
The way I saw it when implementing this, there was no reason to make vanilla a separate thing, since its one and only level will always be the first postcard when not filtering.

It also has the benefit of keeping the code simpler, rather than adding some kind of flag or dummy element in `modsWithLevels`